### ISSUE TITLE
Enum type support without extra configuration

### ISSUE THREAD

**Issue by jakobkmar** [realm/realm-kotlin#829]
Apart from the planned type adapters, could enums be a generally supported type (by saving their name as a string)?

**Comment by rorbech** [comment]
Yes, this should be possible to add to our internal type adapters. We will try to add this.

**Comment by mohit-sharma-87** [comment]
Can we have this, please? It makes my code complicated for simple stuff.

**Comment by sirambd** [comment]
What is the status of the internal type adapter for enums?
